imap_processing/tests/mag/test_mag_l1b.py

```diff
@@ -60,7 +63,12 @@ def test_mag_processing(mag_l1a_dataset):
     np.testing.assert_allclose(
         mag_l1b["vectors"][0].values, [2.2972, 2.2415, 2.2381, 0], atol=1e-4
     )
-    np.testing.assert_allclose(mag_l1b["vectors"][1].values, [0, 0, 0, 0])
+    np.testing.assert_allclose(
```

Comment:
Maybe have the test code calculate these explicitly, or just add a comment for how they're calculated?